lib/internal/test_runner/runner.js

```diff
@@ -513,6 +517,10 @@ function run(options = kEmptyObject) {
     validateArray(globPatterns, 'options.globPatterns');
   }
 
+  if (cwd != null) {
```

Comment:
Since we have a default value for `cwd` do we need the `if` statement here?

Reply:
You're right 😁